auth/src/main/java/org/openedx/auth/presentation/signup/SignUpFragment.kt

```diff
@@ -104,7 +104,10 @@ import org.openedx.core.ui.windowSizeValue
 class SignUpFragment : Fragment() {
 
     private val viewModel by viewModel<SignUpViewModel> {
-        parametersOf(requireArguments().getString(ARG_COURSE_ID, ""))
+        parametersOf(
+            requireArguments().getString(ARG_COURSE_ID, ""),
+            requireArguments().getString(ARG_INFO_TYPE, "")
```

Comment:
We are using empty string here and null in the SignIn screen. Later on in MainFragment, we are only adding the check for the null type. Please either use the same approach or update the MainFragment.